src/formio-component.component.ts

```diff
@@ -85,7 +85,16 @@ export class FormioComponentComponent<T> implements OnInit {
         if (this.component.input && this.component.key) {
             let control = component.getControl();
             if (control) {
-                if (this.component.multiple) {
+                //@TODO: Distinguish between multiple value components and component of components...
+                if (this.component.multiple &&
+                    (this.component.type == 'select' ||
```

Comment:
I don't like how this works since any time we add a new component that meets this criteria, we will just need to "know" to add it here as well. Is there any way to bring this back to the base component as a flag and the classes would set that flag?